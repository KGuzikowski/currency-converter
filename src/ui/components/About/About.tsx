import React from 'react'
import { createStyles, makeStyles, Theme } from '@material-ui/core/styles'
import Typography from '@material-ui/core/Typography'

const useStyles = makeStyles((theme: Theme) =>
  createStyles({
    title: {
      flexGrow: 1,
      textAlign: 'center',
      fontWeight: 'normal',
      marginBottom: theme.spacing(5),
      marginTop: theme.spacing(3)
    }
  })
)

const About = () => {
  const classes = useStyles()

  return (
    <Typography variant="h6" className={classes.title}>
        You can use this tool to search for many different currencies and their historical data.
    </Typography>
  )
}

export default About