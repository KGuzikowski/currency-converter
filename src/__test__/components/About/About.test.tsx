import React from 'react'
import { create } from 'react-test-renderer'
import About from '../../../ui/components/About/About'

it('renders About component correctly', () => {
    const about = create(<About />)
    expect(about.toJSON()).toMatchSnapshot()
})