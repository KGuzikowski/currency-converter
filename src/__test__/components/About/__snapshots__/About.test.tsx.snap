// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders About component correctly 1`] = `
<h6
  className="MuiTypography-root makeStyles-title-1 MuiTypography-h6"
>
  You can use this tool to search for many different currencies and their historical data.
</h6>
`;
